docs/cover.md

```diff
@@ -1,57 +1,71 @@
-# Code coverage for GoAWK
 
-GoAWK implements code coverage functionality, similar to the one built in the Golang itself (https://go.dev/blog/cover).
+# GoAWK's code coverage feature
 
-![screenshot](cover.png)
+GoAWK implements code coverage functionality, similar to the [code coverage built into Go](https://go.dev/blog/cover). This feature was implemented by Volodymyr Gubarkov ([xonixx](https://github.com/xonixx)), primarily in [PR #154](https://github.com/benhoyt/goawk/pull/154).
 
-## How to use
+Here is a screenshot using GoAWK's coverage feature on a simple AWK program (`prog.awk`):
+
+![Example code coverage screenshot](cover.png)
 
-To generate coverage report:
-```
-goawk -f a.awk -coverprofile c.out -covermode set    
-```
-This generates `c.out` file with coverage profile data for the `a.awk` program execution.
-          
-Now it's time to visualize the coverage report `c.out` by rendering it to HTML.
 
-Please note. The code coverage functionality of GoAWK relies on Golang toolset.
-So to generate the HTML coverage report (like on the screenshot above) use the command below. This opens a web browser displaying annotated source code:
+## Basic usage
+
+The simplest way to generate a coverage report is to run your AWK program with the `-coverprofile` option. To run the program in the default coverage mode and write the coverage report to `cover.out`, run the following:
+
 ```
-go tool cover -html=c.out
+$ goawk -f prog.awk -coverprofile cover.out
+will always run
+should run
 ```
 
-Write out an HTML file instead of launching a web browser:
+This generates a file `cover.out` with coverage profile data for the execution of `prog.awk`.
+
+We rely on the Go tool chain to visualize the coverage report, specifically the `go tool cover` command. This command renders a coverage report to HTML. If you don't have Go installed, [install it now](TODO).
+
+To write the HTML coverage report (like the one shown in the screenshot above) to a temporary file and open a web browser displaying it, run the following:
+
 ```
-go tool cover -html=c.out -o coverage.html
+$ go tool cover -html=cover.out
 ```
 
-Please note. At the moment it's not possible to generate coverage report for AWK functions, similar to what is possible for Go:
```

Comment:
I don't think it's necessary to say this here. It's noted in "Future work" below.

Reply:
Agree.